client/src/pages/HomePage.jsx

```diff
@@ -1,10 +1,127 @@
+import { useEffect, useState } from 'react';
+
+import { env } from '../../config/env';
+
+import axios from 'axios';
+
 import Navbar from '../components/Navbar';
 
+const userID = '6737bd5921b1fac154eadf76';
+
+import './mycss.css';
```

Comment:
```suggestion
// import './mycss.css';
```
...or remove mycss.css from the .gitignore; otherwise, this will crash everyone's front end.